fdbclient/include/fdbclient/TenantManagement.actor.h

```diff
@@ -546,6 +546,41 @@ Future<std::vector<std::pair<TenantName, int64_t>>> listTenants(Reference<DB> db
 	});
 }
 
+ACTOR template <class Transaction>
+Future<std::vector<std::pair<TenantName, int64_t>>> listTenantGroupMetadataTransaction(Transaction tr,
```

Comment:
These aren't getting the metadata:

```suggestion
Future<std::vector<std::pair<TenantName, int64_t>>> listTenantGroupTenantsTransaction(Transaction tr,
```